src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -802,7 +912,7 @@ int tap_esc_start(void)
 
 	if (tap_esc == nullptr) {
 
-		tap_esc = new TAP_ESC;
+		tap_esc = new TAP_ESC(_supported_channel_count);
```

Comment:
`initialize_uart` is called from `task_main`, which runs on the same thread as the driver main loop. So this is correct (and why it works), but the naming & startup sequence is really confusing.